specs/CustomScrollbar.md

```diff
@@ -0,0 +1,93 @@
+Custom Scrollbar Style
+===
+
+# Background
+Developers would like to be able to set the scrollbar to Windows fluent overlay style and 
+match the Windows theme. Some customers have used browser flags but they change names and behavior and are not reliable. Thus we want
+to provide an extendable API that allows developers to set scrollbar style. Currently, 
+only Windows fluent overlay are supported, adding new styles in the future will not introduce
+any breaking changes.
+
+# Examples
+## WinRT and .NET   
+```c#
+/// Create WebView Environment with option
+async void CreateEnvironmentWithOption()
+{
+    CoreWebView2EnvironmentOptions options = new CoreWebView2EnvironmentOptions();
+    options.CustomScrollbarStyle = ScrollbarStyle.WindowOverlay;
```

Comment:
Wondering about the word "Style". That word can be confused with CSS style and Xaml style. Is it literally triggering a CSS style?

Reply:
This is not associated with CSS. Happy to rename if helpful.

And actually there are preview CSS styles that modify the scrollbar. We should explicitly document that those are applied on top of whatever native scrollbar styling is selected with this API.

Reply:
Keep name but please update refdocs to note about CSS for scrollbars applying on top.